### PR TITLE
HTTP error for new fsspec

### PR DESCRIPTION
**Pull request by martindurant**
- [x] Tests added / passed
- [x] Passes `black dask` / `flake8 dask`

### CODE REVIEW

File: dask/bytes/tests/test_http.py [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 files = ["a", "b"]
 requests = pytest.importorskip("requests")
+errs = (requests.exceptions.RequestException,)
+if LooseVersion(fsspec.__version__) > "0.7.4":
+    aiohttp = pytest.importorskip("aiohttp")
```
Comment (quasiben) [inline dask/bytes/tests/test_http.py:17]: If we do this and aiohttp is not available won't this skip all the tests?

Comment (martindurant) [reply dask/bytes/tests/test_http.py:17]: Yes, but it really is required for HTTP on fsspec for the release coming soon

Comment (martindurant) [reply dask/bytes/tests/test_http.py:17]: Ah, but maybe you mean that aiohttp should be in the test env .. a good point.

**Comment by martindurant** [comment]
OK, @quasiben ?

**Comment by quasiben** [comment]
Yup, all good!

**Pull request merged**